doc/qutebrowser.1.asciidoc

```diff
@@ -130,7 +130,7 @@ show it.
 == FILES
 
 - '~/.config/qutebrowser/qutebrowser.conf': Main config file.
-- '~/.config/qutebrowser/quickmarks': Saved quickmarks.
+- '~/.local/share/qutebrowser/quickmarks': Saved quickmarks.
```

Comment:
Can you change the order so that the `~/.config/` and `~/.local/share/` paths are not interleaved?